### PR TITLE
[cisco_ftd] Change source.bytes and destination.bytes to long

### PR DESCRIPTION
**Pull request by LaZyDK**
## What does this PR do?

Fixed the data types for source.bytes and destination.bytes.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Relates #6680

### CODE REVIEW

**Review by efd6 — COMMENTED** [review]
Can we have a test case that would overflow otherwise?

**Comment by LaZyDK** [comment]
> Can we have a test case that would overflow otherwise?

Surely. It is now added to the PR.

File: packages/cisco_ftd/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.13.2"
+  changes:
+    - description: Fix source.bytes and destination.bytes type
+      type: bugfix
+      link: https://github.com/elastic/integrations/pull/6680
```
Comment (efd6) [inline packages/cisco_ftd/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/6929
```

Comment (LaZyDK) [reply packages/cisco_ftd/changelog.yml]: Well spotted. Copied the wrong link.

**Review by efd6 — APPROVED** [review]
Thanks

**Pull request merged**